client/blocks/author-selector/switcher-shell.jsx

```diff
@@ -178,8 +179,11 @@ class AuthorSwitcherShell extends React.Component {
 		this.props.updateSearch( false );
 	};
 
-	renderAuthor = author => {
+	renderAuthor = rawAuthor => {
+		const { transformAuthor } = this.props;
+		const author = transformAuthor ? transformAuthor( rawAuthor ) : rawAuthor;
 		const authorGUID = this.getAuthorItemGUID( author );
+
 		return (
 			<PopoverMenuItem
 				className="author-selector__menu-item"
```

Comment:
redundant `bind` on the following line.

Reply:
I agree we could remove the `bind` here, but let's keep such changes outside of already large PRs. We've tried to keep the changes minimal here, so they're easier to review.